core/win32/os.c

```diff
@@ -2497,6 +2497,15 @@ static bool
 os_take_over_thread(dcontext_t *dcontext, HANDLE hthread, thread_id_t tid, bool suspended)
 {
     bool success = true;
+
+    if (DYNAMO_OPTION(skip_terminating_threads)) {
+        if (nt_is_thread_terminating(hthread)) {
+            // takeover fails when attaching and trying to takeover terminating threads
+            // luckily, we don't really need to take over them
```

Comment:
style nit: Use capitalized first words and end punctuation for multi-sentence comments (some old code does not have this new style convention but we prefer it for all new code)

Reply:
Done